package.json

```diff
@@ -47,15 +47,17 @@
     "@types/node": "20.3.1",
     "@types/react": "18.2.8",
     "@types/react-dom": "18.2.5",
-	"@typescript-eslint/eslint-plugin": "^7.6.0",
-	"@typescript-eslint/parser": "^7.6.0",
+    "@typescript-eslint/eslint-plugin": "^7.6.0",
+    "@typescript-eslint/parser": "^7.6.0",
     "encoding": "^0.1.13",
+    "eslint": "^9.0.0",
+    "eslint-config-next": "14.1.4",
     "eslint-config-prettier": "^9.1.0",
-	"eslint-config-next": "14.1.4",
-	"eslint-plugin-jsx-a11y": "^6.8.0",
+    "eslint-plugin-jsx-a11y": "^6.8.0",
     "jest": "^29.7.0",
     "jest-environment-jsdom": "^29.7.0",
     "prettier": "3.2.5",
+    "prettier-plugin-tailwindcss": "^0.5.14",
```

Comment:
I'm assuming this is used to help the implementation for Components?

Reply:
This is used to automatically sort the tailwind classes with prettier to avoid any CSS issues. This should have been part of my original Prettier issue, but I suppose I neglected to install it.